### PR TITLE
[MOO-1597] Update app permissions

### PR DESCRIPTION
**Pull request by Ahmad-Elsayed**

### CODE REVIEW

**Comment by ConnorLand** [comment]
This should be ready to merge around Sep 18. Ahmed will reach out to the team via #docs.

**Comment by ConnorLand** [comment]
This should be ready to merge around Sep 18. Ahmed will reach out to the team via #docs.

File: content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md [hunk 1]
```diff
@@ -157,6 +154,65 @@ To allow a Siri permission you will need to enable Siri capability in your app.
 	</plist>
     ```
 
+#### Execute Pod Install in your iOS directory
+
+Next, execute `pod install` in your iOS app's directory to _______.
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: @Ahmad-Elsayed can you fill in the blank here to make it clear what running `pod install` will do/enable?

Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: done

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**